tests/specs/send/send-stx.spec.ts

```diff
@@ -233,7 +233,7 @@ test.describe('send stx: tests on mainnet', () => {
         .innerText();
       const fee = Number(feeToBePaid.split(' ')[0]);
       // Using min/max fee caps
-      const isMiddleFee = fee >= 0.003 && fee <= 0.75;
+      const isMiddleFee = fee === 0.0004;
       test.expect(isMiddleFee).toBeTruthy();
```

Comment:
```suggestion
      test.expect(isMiddleFee).toEqual(true);
```

is a better test here